lib/screens/monster/monster_info_subtab.dart

```diff
@@ -1110,6 +1110,16 @@ class AwokenSkillSection extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    // Create map to merge multiple instances of same awakening
+    var map = Map();
+    _awakenings.forEach((awakening) {
```

Comment:
Suggest a slight tweak:
```
var map = LinkedHashMap();
_awakenings.map((a) => a.awokenSkill).forEach((as) => map[as] = (map[as] ?? 0) + 1);
```

Reply:
I really need to understand lambda functions haha, I'll make the change